_test/test_utilities_herbert/test_str_to_iarray.m

```diff
@@ -0,0 +1,187 @@
+classdef test_str_to_iarray < TestCase
+    % Test test_str_to_iarray
+    %
+    % str_to_iarray converts a string, character array or cell array of strings
+    % to an integer array
+    
+    methods
+        %--------------------------------------------------------------------------
+        % Single number
+        %--------------------------------------------------------------------------
+        function test_single_number (~)
+            % Test constructor with no arguments
```

Comment:
what is it constructing? why are there no args?

Reply:
COmment deleted and useful one added. And in a dozen or so similar places